src/common/transformations/tests/op_conversions/convert_sequences_to_ti_test.cpp

```diff
@@ -194,12 +194,12 @@ TEST(TransformationTests, ConvertLSTMSequenceToTensorIteratorDynamic) {
 
         auto Ho = std::make_shared<opset5::Result>(rnn_cell->output(0));
 
+        auto Co = std::make_shared<opset5::Result>(rnn_cell->output(1));
```

Comment:
Just curios is cosmetic or change something?

Reply:
It's intentional. Order of node creation impacts on default unique name of the node, what is crucial for graph comparator to set proper inputs order (same for both being compared subgraph nodes).